src/features/data-import/manual-entry/data-input.tsx

```diff
@@ -177,14 +186,25 @@ Coins per hour	860.06B"
             )}
           </ResponsiveDialogBody>
 
-          <ResponsiveDialogFooter mobileLayout="1-2">
+          <ResponsiveDialogFooter mobileLayout="equal">
             <Button 
               variant="outline" 
               onClick={closeModal}
               className="h-10 hover:bg-destructive/10 hover:text-destructive hover:border-destructive/50 transition-all duration-200"
             >
               Cancel
             </Button>
+            <Button 
```

Comment:
These are the changes from [PR 168](https://github.com/jarekb84/TowerOfTracking/pull/168), please merge that PR and rebase this branch to help isolate the changes.